src/docs/sphinx/developerGuide/Contributing/index_contributing.rst

```diff
@@ -9,8 +9,6 @@ Contributing
 
    GitWorkflow.rst
 
-   developingOnDocker.rst
```

Comment:
I think this link needs to be kept & changed to `UsingDocker.rst`. Otherwise, I do not think you can navigate to the new page.

Reply:
Yes @bmhan12 I hesitated about this because the subject is "_unsupported officially_".
I did not want to give too much visibility, but I still want the possibility to share the link to help anyone who's willing to take the journey.
This may be stupid since it's still available from searching (hopefully...).

What do we want to do? @herve-gross @joshua-white

Reply:
IMHO, we should make this information accessible.